src/styles/expensiTextInputPlatformStyles/index.ios.js

```diff
@@ -0,0 +1,3 @@
+export default {
+    paddingTop: 19,
+};
```

Comment:
From https://github.com/Expensify/App/issues/6390#issuecomment-991894144.
_We should try to avoid using styles specific to platforms. Try to generalize the styles so that they look good on each platform._

This change affects only IOS creating inconsistency across platforms. Also, this is a big design decision and thus I will ask someone to review this.

Reply:
The default padding top is 23, I'll move it to the file for clarity. @parasharrajat

Reply:
Also, I had to do similar in another issue related to emojis

Reply:
https://github.com/Expensify/App/pull/6289

Reply:
I agree, I forgot to mention that at that time but there, UI was not affected. Text as vertically aligned on all platforms and here the gap is not equal.

Reply:
Can you help me understand what this change does, and how it helps us fix the issue that is being addressed here?

Reply:
@parasharrajat gotcha

@shawnborton long story short, emojis on iOS display a bit larger ( https://github.com/Expensify/App/issues/6390#issuecomment-983973527 ) and require some extra pixels to display it properly. The input has `height: 100%; paddingBottom: 8; paddingTop: 23`, and the parent has `height: 50`, so, the usable height for the input is `19`, but as the input has `fontSize: 15`, it needs 4 extra pixels to render the emoji properly, that's why changing `paddingTop: 23` to `paddingTop: 19` address the issue.

We can also use `paddingBottom: 6; paddingTop: 21` to try to keep it a bit more at center...